docs/pages/database-access/guides/rds.mdx

```diff
@@ -18,99 +18,348 @@ This guide will help you to:
+  databases. 
+
 - (!docs/pages/includes/tctl.mdx!)
 
+If you plan to run the Teleport Database Service on Kubenetes, you will need the
+following:
+
+ - The `aws` CLI in your PATH. Install it by following the [AWS
+   documentation](https://aws.amazon.com/cli/).
+ - An IAM OIDC provider running in your Kubernetes cluster. See the [AWS
+   documentation](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html)
+   for how to create an IAM OIDC provider.
+
+   To check whether you have an IAM OIDC provider running in your cluster, run
+   the following `aws` command, assigning <Var name="eks-region" /> to the
+   region where your EKS cluster is running and <Var name="cluster-name" /> to
+   the name of your Kubernetes cluster:
+
+   ```code
+   $ aws --region=<Var name="eks-region" /> eks describe-cluster --name <Var name="cluster-name"/> --query "cluster.identity.oidc.issuer" --output text
+   ```
+
+   If you have an IAM OIDC provider associated with your cluster, this command
+   will print its ID.
+
+ - The [`jq` CLI](https://jqlang.github.io/jq/), which we use to process JSON
+   data in this guide.
+
 ## Step 1/6. Create a Teleport user
 
 (!docs/pages/includes/database-access/create-user.mdx!)
 
 ## Step 2/6. Create a Database Service configuration
 
-(!docs/pages/includes/database-access/token.mdx!)
+In this section, you will configure the Teleport Database Service. To do so, you
+will:
+
+- Create a join token for the service to demonstrate trust with your Teleport
+  cluster
+- Set up your package manager so you can install and run the Database Service
+- Generate a configuration for the Database Service
+
+### Create a join token
+
+Establish trust between the Teleport Database Service and your Teleport cluster
+by creating a join token.
+
+Generate a join token by running the following command on your workstation: 
+
+```code
+$ tctl tokens add --type=db
+```
+
+The next step depends on how you plan to run the Teleport Database Service:
+
+<Tabs>
+<TabItem label="Linux Server">
+
+Save the token in a file called `/tmp/token` on the host that will run the
+Database Service.
+
+</TabItem>
+<TabItem label="Kubernetes">
+
+Later in this guide, you will use this join token when configuring the Teleport
+Database Service.
+
+</TabItem>
+</Tabs>
 
 (!docs/pages/includes/database-access/alternative-methods-join.mdx!)
 
+### Prepare your environment
+
+Next, get your environment ready to run the Teleport Database Service:
+
+<Tabs>
+<TabItem label="Linux Host">
+
 Install Teleport on the host where you will run the Teleport Database Service:
 
 (!docs/pages/includes/install-linux.mdx!)
 
-Create the Database Service configuration:
+Generate a configuration for the Teleport Database Service:
 
-<ScopedBlock scope={["oss", "enterprise"]}>
+Run the command below on your Linux host. Assign <Var
+name="mytenant.teleport.sh:443" /> to the host **and port** of your Teleport
```

Comment:
Xin said something in another PR about moving away from the term `tenant`, so you might wanna replace `mytenant` with `example` or similar.



```suggestion
name="example.teleport.sh:443" /> to the host **and port** of your Teleport
```